src/js/components/Panel/Panel.js

```diff
@@ -39,6 +44,7 @@ Panel.propTypes = {
   children: PropTypes.node.isRequired,
   isStriped: PropTypes.bool,
   maxSize: PropTypes.string,
+  panelImage: PropTypes.object,
```

Comment:
I wonder if it's worth calling out what keys the panelImage object has.
`
PropTypes.shape({Source, PlaceHolder})
`
?

Reply:
Oh yeah I like that, that's much clearer